core/src/test/java/de/jplag/special/ReadmeCodeExampleTest.java

```diff
@@ -35,10 +36,12 @@ void testReadmeCodeExample() {
             JPlagResult result = JPlag.run(options);
 
             // Optional
-            ReportObjectFactory reportObjectFactory = new ReportObjectFactory();
-            reportObjectFactory.createAndSaveReport(result, "/path/to/output");
+            ReportObjectFactory reportObjectFactory = new ReportObjectFactory(new File("/path/to/output"));
```

Comment:
This test is meant to remind us that we need to adapt the readme if we change the basic usage of JPlag!